### ISSUE TITLE
Style in Theme is recommended to default to Follow system light or dark

### ISSUE THREAD

**Issue by p5ydn0** [zombieFox/nightTab#200]
Thank you for sharing, I like it very much.

**Style** in **Theme** is recommended to default to `Follow system light or dark`. I think this way will let more people choose.

Thank you.

**Comment by zombieFox** [comment]
> Thank you for sharing, I like it very much.

Thank you!

> Style in Theme is recommended to default to Follow system light or dark. I think this way will let more people choose.

nightTab does have controls to let users choose a light, dark or auto theme. The default setting on a fresh install is "dark".

Do you mean making the default Theme Style "auto" so it follows the OS system theme?

**Comment by p5ydn0** [comment]
> > Thank you for sharing, I like it very much.
> 
> Thank you!
> 
> > Style in Theme is recommended to default to Follow system light or dark. I think this way will let more people choose.
> 
> nightTab does have controls to let users choose a light, dark or auto theme. The default setting on a fresh install is "dark".
> 
> Do you mean making the default Theme Style "auto" so it follows the OS system theme?

Thank you for sharing and replying.

Yes, I mean that. I think the setting most users will prefer is `auto`; because opening a new tab under a dark theme will be a bit "dazzling".

This is just my personal opinion. Thank you.

**Comment by Excigma** [comment]
Some people are using light mode for their operating system due to accessibility concerns, this might also slightly help them

**Comment by zombieFox** [comment]
Yup, understood and good points. However the project is a "dark themed" start page, so comes preset with some "opinionated design choices" out of the box. These choices can be changed in the settings so I don't see a lot of value in changing the default options. If anything nightTab would be come more generic in doing so. 

Personally I feel the default settings works to distinguish nightTab from the crowd of other start page projects while allowing users to change most all visual elements.

**Comment by zombieFox** [comment]
Closing as no change or bug fix is needed.
